demos/checkboxradio/product-selector.html

```diff
@@ -0,0 +1,121 @@
+	<title>jQuery UI Checkboxradio - Product Selector</title>
+	<link rel="stylesheet" href="../../themes/base/all.css">
+	<link rel="stylesheet" href="../demos.css">
+	<script src="../../external/requirejs/require.js"></script>
+	<script src="../bootstrap.js" data-modules="controlgroup">
+		$( "input" ).checkboxradio();
+		$( "[name='shape']").on( "change", function(){
+			$( ".shape" ).removeClass( "circle pill square rectangle" )
+				.addClass( $( this ).val() );
+		});
+		$( ".toggle" ).on( "change", function(){
+			if ( $( this ).is( ".brand-toggle" ) ) {
+				var checked = $( this ).is( ":checked" ),
+					value = $( "[name='brand']" ).filter( ":checked" ).attr( "data-" + this.id )
+				$( ".shape" ).css( this.id, checked ? value : "" );
+			} else {
+				$( ".shape" ).toggleClass( this.id, $( this ).is( ":checked") );
+			}
+		});
+		$( "[name='brand']").on( "change", function(){
+			$( "input" ).filter( ":checked" ).not( this ).trigger( "change" );
+		});
+		$( "input" ).filter( ":checked" ).trigger( "change" );
+		$( ".shape-bar, .brand" ).controlgroup();
+		$( ".toggles" ).controlgroup({
+			direction: "vertical"
+		});
+	</script>
+	<style>
+		.shape {
+			margin-left: 4em;
+			margin-top: 2.5em;
+			height: 8em;
+			width: 8em;
+			box-shadow: 4px 4px 8px;
+			color: #ccc;
+			background-repeat: no-repeat;
+			background-size: 90%;
+			background-position: 50%;
+		}
+		.circle, .pill {
+			border-radius: 8em;
+		}
+		.pill, .rectangle {
+			width: 16em;
+		}
+		.square, .circle {
+			margin-left: 9em;
+		}
+		.border {
+			border: 2px solid #333333;
+		}
+		.toggles {
+			width: 200px;
+		}
+		.toggle-wrap, .shape {
+			display: inline-block;
+			vertical-align: top;
+		}
+		.controls {
+			background: #ccc;
+			padding: 1em;
+			display: inline-block;
+		}
+	</style>
+</head>
+<body>
+<div class="controls">
+	<div class="brand-wrap">
+		<h3>1.) Select a brand</h3>
+		<div class="brand">
+			<label for="brand-jquery">jQuery</label>
+			<input data-background-color="#0769AD" data-color="#7ACEF4" data-background-image="url(images/jquery.png)" type="radio" name="brand" id="brand-jquery">
+			<label for="brand-jquery-ui">jQuery UI</label>
+			<input data-background-color="#B24926" data-color="#FAA523" data-background-image="url(images/jquery-ui.png)" type="radio" name="brand" id="brand-jquery-ui" checked>
+			<label for="brand-jquery-mobile">jQuery Mobile</label>
+			<input data-background-color="#108040" data-color="#3EB249" data-background-image="url(images/jquery-mobile.png)" type="radio" name="brand" id="brand-jquery-mobile">
+			<label for="brand-sizzle">Sizzle</label>
+			<input data-background-color="#9A1B1E" data-color="#FAA523" data-background-image="url(images/sizzle.png)" type="radio" name="brand" id="brand-sizzle">
+			<label for="brand-qunit">QUnit</label>
+			<input data-background-color="#390F39" data-color="#9C3493" data-background-image="url(images/qunit.png)" type="radio" name="brand" id="brand-qunit">
```

Comment:
Move `type`, `name`, and `id` to the beginning, potentially split the data attributes on to a separate line.